StringSearcher.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Linq;
+
+namespace Task5
+{
+    public class StringSearcher
+    {
+        public string[] Search(string[] array)
+        {
+            return array.Where(s => s.Length > 0)
+                .Select(s => s.Substring(Math.Min(3, s.Length)).ToUpper())
```

Comment:
Метод должен оставлять только первые 3 символа строки, а не удалять!
 ```suggestion
                .Select(s => s.Substring(0, Math.Min(3, s.Length)).ToUpper())
```